core/rest/src/main/java/org/trafodion/rest/util/TrafLdap.java

```diff
@@ -0,0 +1,94 @@
+package org.trafodion.rest.util;
+
+import java.util.Hashtable;
+
+import javax.naming.Context;
+import javax.naming.NamingException;
+
+import javax.naming.ldap.InitialLdapContext;
+import javax.naming.ldap.LdapContext;
+
+public class TrafLdap {
+
+    private String ip = "10.10.10.3";
+    private String port = "389";
+    private String basedn = "cn=users,cn=accounts,dc=esgyncn,dc=local";
+    private String username="trafodion";
+    private String password="traf123";
+

```

Comment:
Even for test code, it seems this type of info should be in a config file, not hard-coded.